tests/lambda_endpoints/test_path_helper.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from sds_data_manager.lambda_code.SDSCode.path_helper import FilenameParser
+
+
+def test_date_checker():
+    """Tests date inputs"""
+    filename = "imap_glows_l0_raw_20231010_20231011_v01-01.pkts"
+    file_parser = FilenameParser(filename)
+    assert file_parser.check_date_input("20200101")
```

Comment:
Why do you need to input a date to the fileparser? Shouldn't this be acting on `self.date`?